docs/source/faq.rst

```diff
@@ -97,3 +97,38 @@ For instance, you can stop showing each trial result as follows:
 
 
 Please refer to :class:`optuna.logging` for further details.
+
+
+How does Optuna handle NaNs and errors reported by the objective function?
```

Comment:
How about using `exceptions` instead of `errors`? It may make it clearer.

Reply:
As you mentioned, what I meant here is `exception`. I've fixed it. Thank you.